docs/configuration.md

```diff
@@ -15,7 +17,7 @@ It's recommended that you also set either `EMPIRE_GITHUB_ORGANIZATION`, or `EMPI
 
 ### SAML Authentication
 
-Refer to the [docs](./saml) on configuring the SAML authentication backend.
+Refer to the [docs](./saml.md) on configuring the SAML authentication backend.
```

Comment:
I've added SAML to mkdocs.yml now. And from what I can see in https://github.com/remind101/empire/blob/master/docs/quickstart_using.md#empire--using ... it looks like you still need an `.md` (looking at the installing link)